server/src/controllers/messages/index.ts

```diff
@@ -19,7 +19,7 @@ export function postMessage(req: Request, res: Response) {
 
 export function getMessages(req: Request, res: Response) {
   const username = (req.user as any).username
-  const { roomId } = req.body
+  const roomId = req.query.roomId.toString()
```

Comment:
Should check if `req.query.roomId` isn't undefined.If it is, return appropriate response with error code 422

Reply:
Yep, I had thought about this - will need to add param checks for all the API functions.